addon/components/au-icon.js

```diff
@@ -18,4 +18,16 @@ export default class AuButton extends Component {
     if (this.args.alignment == 'right') return 'au-c-icon--right';
     else return '';
   }
+
+  get classAttribute() {
+    return `au-c-icon au-c-icon--${this.args.icon} ${this.alignment} ${this.size}`;
+  }
+
+  get ariaHiddenAttribute() {
+    if (!this.args.ariaHidden || this.args.ariaHidden === 'false') {
```

Comment:
just a small note, I don't think this behaviour is entirely equal to the previous one. make sure to test with undefined and null values for ariaHidden.

Reply:
ah right, this will probably need an explicit comparison with `false`